cms/static/js/views/utils/view_utils.js

```diff
@@ -173,6 +174,33 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
             return false;
         };
 
+        keywordValidator = (function () {
+            var regexp = /%%+[^%]+%%/g;
+            var keywords = ['%%USER_ID%%', '%%USER_FULLNAME%%', '%%COURSE_DISPLAY_NAME%%', '%%COURSE_END_DATE%%'];
+            var validate = function (string) {
```

Comment:
This should be a named function.

Named functions are defined at compile time, whereas anonymous functions assigned to variables are defined at runtime, among other subtleties [1].

An unsung benefit to named functions over anonymous functions is that they provide a more explicit stack trace when debugging.

```javascript
function validate(string) {
    // ...
}
```

[1] http://stackoverflow.com/questions/336859/var-functionname-function-vs-function-functionname